themes/bootprint3/less/search.less

```diff
@@ -202,3 +202,10 @@ legend {
     }
   }
 }
+
+.backtoorigin > p > a {
```

Comment:
I'd prefer `back-to-origin` (using it also for the template file name would make it easier to read).